Tasks/XamarinTestCloud/XamarinTestCloud.ps1

```diff
@@ -187,8 +187,8 @@ if($testCloudResults)
     $mdReportFile = Join-Path $testDir "xamarintestcloud_$buildId.md"
     foreach($result in $testCloudResults)
     {
-       Write-Output $result | Out-File $mdReportFile -Append
-       Write-Output [Environment]::NewLine | Out-File $mdReportFile -Append
+       Write-Output "[$result]($result)" | Out-File $mdReportFile -Append
```

Comment:
Is this a link?  You can just use angle brackets: `<$result>`; as in `<https://www.microsoft.com/en-us/>`

Reply:
The new markdown parser that is being used does not accept that format for links. The only valid format is the [name](URL).